db/migrations/0002_rename_name_player_nickname_alter_guild_description.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.0.2 on 2023-11-24 18:52
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='player',
+            old_name='name',
+            new_name='nickname',
+        ),
+        migrations.AlterField(
+            model_name='guild',
+            name='description',
+            field=models.TextField(null=True),
+        ),
+    ]

```

Comment:
It is a good practice to include one migration per app in one pull request. Add one migration with all required changes.